hack/e2e.go

```diff
@@ -81,12 +82,21 @@ func main() {
 }
 
 func wait(cmd string, args ...string) error {
+	sig_channel := make(chan os.Signal, 1)
+	signal.Notify(sig_channel, os.Interrupt)
+
 	c := exec.Command(cmd, args...)
 	c.Stdout = os.Stdout
 	c.Stderr = os.Stderr
 	if err := c.Start(); err != nil {
 		return err
 	}
+	go func() {
+		sig := <-sig_channel
+		if err := c.Process.Signal(sig); err != nil {
+			log.Fatalf("err: %v", err)
```

Comment:
Please make the error message easier for a human to understand the context, something like
`Fatalf("could not send %s signal %s: %v", cmd, sig, err)`